### ISSUE TITLE
A strange detection result. High MAP but wrong detection bbox

### ISSUE THREAD

**Issue by 315386775** [rbgirshick/py-faster-rcnn#601]
After training my own model.  I tested the model and get **the result:**

```
im_detect: 1/5 0.726s 0.000s
im_detect: 2/5 0.651s 0.001s
im_detect: 3/5 0.617s 0.001s
im_detect: 4/5 0.606s 0.001s
im_detect: 5/5 0.596s 0.000s
Evaluating detections
Writing nucel VOC results file
VOC07 metric? Yes
Reading annotation for 1/5
Saving cached annotations to /home/xxx/py-faster-rcnn/data/VOCdevkit2007/annotations_cache/annots.pkl
AP for cat = 0.8894
Mean AP = 0.8894
~~~~~~~~
Results:
0.889
0.889
~~~~~~~~
```
And the related loss is that:
```
Iteration 69960 (0.346406 iter/s, 57.7357s/20 iters), loss = 0.1463
Train net output #0: bbox_loss = 0.119585 (* 1 = 0.119585 loss)
Train net output #1: cls_loss = 0.015627 (* 1 = 0.015627 loss)
Train net output #2: rpn_cls_loss = 0.000290064 (* 1 = 0.000290064 loss)
Train net output #3: rpn_loss_bbox = 0.00230009 (* 1 = 0.00230009 loss)
Iteration 69960, lr = 0.0001
Iteration 69980 (0.351813 iter/s, 56.8483s/20 iters), loss = 0.147291
Train net output #0: bbox_loss = 0.110938 (* 1 = 0.110938 loss)
Train net output #1: cls_loss = 0.106108 (* 1 = 0.106108 loss)
Train net output #2: rpn_cls_loss = 0.000747744 (* 1 = 0.000747744 loss)
Train net output #3: rpn_loss_bbox = 0.00341549 (* 1 = 0.00341549 loss)
Iteration 69980, lr = 0.0001
Wrote snapshot to: /home/xxx/output/faster_rcnn_end2end/voc_2007_trainval/zf_faster_rcnn_iter_70000.caffemodel
done solving
```
But when I test one image used by demo.py. I get the all wrong detection results . The bounding boxes are all wrong. Anyone can help me ? I checked the demo.py. I used the right model and right prototxt.

**Comment by ranju2015** [comment]
You can try with test bounding box regression = False

**Comment by 315386775** [comment]
@ranju2015 thanks. it works!

**Comment by Zhoutyler** [comment]
@ranju2015 Thanks! I met the same problem and your solution helped me out.
